ci-operator/step-registry/ocm/ci/fastforward-multiple/ocm-ci-fastforward-multiple-commands.sh

```
@@ -0,0 +1,444 @@
+    local pr_branch="add-tekton-files-${dest_versions// /-}"
+    git checkout -b "${pr_branch}" 2>&1
+
+    local files_created=false
+
+    # For each destination version
+    for dest_version in ${dest_versions}; do
+      # Convert 5.0 -> 50, 5.1 -> 51
+      local dest_ver_compact="${dest_version//./}"
+
+      # Check if files already exist
+      if ls .tekton/*-${product_prefix}-${dest_ver_compact}-*.yaml >/dev/null 2>&1; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)